src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -63,7 +63,7 @@
     <PackageReference Include="NUnit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.1.1" />
-    <PackageReference Include="FirebirdSql.Data.FirebirdClient" Version="6.3.0" />
+    <PackageReference Include="FirebirdSql.Data.FirebirdClient" Version="6.6.0" />
```

Comment:
Maybe just update to the same version as in master.

Reply:
But it's the same as in master:
https://github.com/nhibernate/nhibernate-core/blob/master/src/NHibernate.Test/NHibernate.Test.csproj#L66

I just cherry-picked your commit https://github.com/nhibernate/nhibernate-core/commit/2b8dd9be20e3f35a4e609f1f41e6333bcf36501e in the end.

Reply:
But TeamCity FireBird is failing..
```C#
FirebirdSql.Data.FirebirdClient.FbException : I/O error during "CreateFile (create)" operation for file "D:\nhibernate.fdb"
Error while trying to create file
  ----> FirebirdSql.Data.Common.IscException : I/O error during "CreateFile (create)" operation for file "D:\nhibernate.fdb"
Error while trying to create file
   at FirebirdSql.Data.FirebirdClient.FbConnection.CreateDatabaseImpl(String connectionString, Int32 pageSize, Boolean forcedWrites, Boolean overwrite)
   at FirebirdSql.Data.FirebirdClient.FbConnection.CreateDatabase(String connectionString, Int32 pageSize, Boolean forcedWrites, Boolean overwrite)
   at NHibernate.TestDatabaseSetup.DatabaseSetup.SetupFirebird(Configuration cfg) in D:\BuildAgent\work\30546188361a242\src\NHibernate.TestDatabaseSetup\TestDatabaseSetup.cs:line 125
   at NHibernate.TestDatabaseSetup.DatabaseSetup.SetupDatabase() in D:\BuildAgent\work\30546188361a242\src\NHibernate.TestDatabaseSetup\TestDatabaseSetup.cs:line 54
--IscException
   at FirebirdSql.Data.Client.Managed.Version10.GdsDatabase.ProcessResponse(IResponse response)
   at FirebirdSql.Data.Client.Managed.Version10.GdsDatabase.ReadResponse()
   at FirebirdSql.Data.Client.Managed.Version13.GdsDatabase.CreateDatabase(DatabaseParameterBuffer dpb, String dataSource, Int32 port, String database, Byte[] cryptKey)
   at FirebirdSql.Data.FirebirdClient.FbConnectionInternal.CreateDatabase(DatabaseParameterBuffer dpb)
   at FirebirdSql.Data.FirebirdClient.FbConnection.CreateDatabaseImpl(String connectionString, Int32 pageSize, Boolean forcedWrites, Boolean overwrite)
------- Stdout: -------
FirebirdSql.Data.FirebirdClient.FbException (0x80004005): lock time-out on wait transaction
object nhibernate is in use ---> lock time-out on wait transaction
object nhibernate is in use
   at FirebirdSql.Data.FirebirdClient.FbConnection.DropDatabase(String connectionString)
   at NHibernate.TestDatabaseSetup.DatabaseSetup.SetupFirebird(Configuration cfg) in D:\BuildAgent\work\30546188361a242\src\NHibernate.TestDatabaseSetup\TestDatabaseSetup.cs:line 116
```

Some problem with firebird service on TeamCity build machine?

Reply:
> Some problem with firebird service on TeamCity build machine

And master now fails the same:
https://teamcity.jetbrains.com/viewLog.html?buildId=3508163&buildTypeId=bt874

Reply:
Can someone restart this build machine?

Reply:
Done

Reply:
Thank. So if TeamCity tests are green it's ready to go..